api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -105,6 +110,245 @@ message RateLimit {
   bool disable_x_envoy_ratelimited_header = 9;
 }
 
+// Global rate limiting :ref:`architecture overview <arch_overview_global_rate_limit>`.
+// Also applies to Local rate limiting :ref:`using descriptors <config_http_filters_local_rate_limit_descriptors>`.
```

Comment:
Please add a `[#not-implemented-hide:]` tag here.

Reply:
Done!